qutebrowser/misc/editor.py

```diff
@@ -46,20 +46,18 @@ class ExternalEditor(QObject):
     def __init__(self, win_id, parent=None):
         super().__init__(parent)
         self._text = None
-        self._oshandle = None
-        self._filename = None
+        self._file = None
```

Comment:
This [broke](https://travis-ci.org/The-Compiler/qutebrowser/jobs/150305323#L668) various tests in `tests/unit/misc/test_editor.py `.